### ISSUE TITLE
Tables interface broken

### ISSUE THREAD

**Issue by andreasnoack** [JuliaData/DataFrames.jl#1711]
```julia
julia> DataFrame([(a = 2, b = 'a')])
ERROR: UndefVarError: DataValueUnwrapper not defined
Stacktrace:
 [1] DataFrame(::Array{NamedTuple{(:a, :b),Tuple{Int64,Char}},1}) at /Users/andreasnoack/.julia/packages/DataFrames/lyCjP/src/other/tables.jl:29
 [2] top-level scope at none:0
```
This happened in https://github.com/JuliaData/Tables.jl/commit/5c897d5c57bd1420173a8ef24dfcc6a389222015 which was released as v0.1.15.

cc: @quinnj

**Comment by nalimilan** [comment]
You realize how JuliaCIBot is useful when it's down...

**Comment by andreasnoack** [comment]
Looks like this is also fixed by #1710
